src/utils/power_flow/power_flow.jl

```diff
@@ -99,8 +99,25 @@ function _write_pf_sol!(sys::System, nl_result)
     return
 end
 
+# TODO: Apply actions according to load type
+function _get_load_data(sys::System, b::Bus)
+    activepower = 0.0
+    reactivepower = 0.0
+    for l in get_components(ElectricLoad, sys)
+        if !isa(l, FixedAdmittance) && (l.bus == b)
+            activepower += get_activepower(l)
+            reactivepower += get_reactivepower(l)
+        elseif isa(l, FixedAdmittance) && (l.bus == b)
```

Comment:
This calls `isa` on l twice and compares the buses twice.  You could easily refactor the code to call them each only once.